### PR TITLE
Added the opendyslexic font

### PR DESCRIPTION
**Pull request by balupton**

### CODE REVIEW

File: Casks/font-opendyslexic.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FAMILY: OpenDyslexic
+cask :v1 => 'font-opendyslexic' do
+  version 'f4b5ba89018b44d633608907e15f93fb3fabbabc'
+  sha256 '579285694acc7dbe007faf7b54e02b4be3f93837c37f5304e0c090e0d36178d1'
+
+  url 'https://github.com/antijingoist/open-dyslexic/archive/f4b5ba89018b44d633608907e15f93fb3fabbabc.zip'
+  homepage 'https://github.com/antijingoist/open-dyslexic'
+  license :bvl
```
Comment (vitorgalvao) [inline Casks/font-opendyslexic.rb]: Not a [valid license](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#valid-licenses).

File: Casks/font-opendyslexic.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FAMILY: OpenDyslexic
```
Comment (vitorgalvao) [inline Casks/font-opendyslexic.rb]: What’s the purpose of this line?

Comment (balupton) [inline Casks/font-opendyslexic.rb]: No clue, the auto generator script added it.

Comment (vitorgalvao) [inline Casks/font-opendyslexic.rb]: Pinging @rolandwalker.

Comment (rolandwalker) [inline Casks/font-opendyslexic.rb]: Hi @balupton !  Thanks for the contribution!

The FAMILY line is just informational, and perhaps should be preceded with a `#` character.

The Cask itself starts with `cask :v1…` as always.

The script is actually by @ndr-qef, but I did mangle it in a few ways.

**Comment by ndr-qef** [comment]
Thanks, and sorry for the inconvenience — the output of the script can be confusing if you are unfamiliar with our DSL. I merged your PR as d52ff67525.

**Pull request closed without merging**